src/PromotedMetricsType.tsx

```diff
@@ -18,67 +29,85 @@ export type PromotedMetricsType = {
    * Starts logging session with the provided user and logs a
    * user event.
    */
-  startSessionAndLogUser(userId: string): void;
+  startSessionAndLogUser(userId: string): void
 
   /**
    * Call when sign-in completes with no user.
    * Starts logging session with signed-out user and logs a
    * user event.
    */
-  startSessionAndLogSignedOutUser(): void;
+  startSessionAndLogSignedOutUser(): void
 
   // Impression logging
 
   /**
    * Logs an impression for given content.
-   * Typically, you would call useImpressionTracker() for use with
+   * Typically, you would call withCollectionTracker() for use with
```

Comment:
Interesting change.  Will clients not useImpressionTracker at all anymore?

Reply:
ImpressionTracker will still be around for cases where we need to use only impression tracking, but will not be the default/recommended choice.